invenio_app_ils/config.py

```diff
@@ -398,9 +402,9 @@ def _(x):
     ),
 )
 
-CIRCULATION_ITEM_RESOLVING_PATH = "/api/resolver/circulation/items/<item_pid>"
+CIRCULATION_ITEM_RESOLVING_PATH = "/api/resolver/loans/<loan_pid>/item"
```

Comment:
Won't it be a bit confusing to remove the `circulation` part out of the link, as in general we use it for the circulation module?

Reply:
True, I will re-add it! 👍

Reply:
it also happens in links before